shark/examples/shark_inference/stable_diffusion/main.py

```diff
@@ -0,0 +1,125 @@
+from transformers import CLIPTextModel, CLIPTokenizer
+import torch
+from PIL import Image
+from diffusers import LMSDiscreteScheduler
+from tqdm.auto import tqdm
+import numpy as np
+from stable_args import args
+from model_wrappers import (
+    get_vae32,
+    get_vae16,
+    get_unet16_wrapped,
+    get_unet32_wrapped,
+)
+from utils import get_shark_model
+
+GCLOUD_BUCKET = "gs://shark_tank/prashant_nod"
+VAE_FP16 = "vae_fp16"
+VAE_FP32 = "vae_fp32"
+UNET_FP16 = "unet_fp16"
+UNET_FP32 = "unet_fp32"
+
+
+def get_models():
+    if args.precision == "fp16":
+        if args.import_mlir == True:
+            return get_vae16(), get_unet16_wrapped()
+        else:
+            return get_shark_model(GCLOUD_BUCKET, VAE_FP16), get_shark_model(
+                GCLOUD_BUCKET, UNET_FP16
+            )
+
+    elif args.precision == "fp32":
+        if args.import_mlir == True:
+            return get_vae16(), get_unet16_wrapped()
```

Comment:
This looks like it should be unet32 and vae32 for fp32.

Reply:
Done.